governance/governance.md

```diff
@@ -0,0 +1,265 @@
+- In our focused case, familiarity with usual open source
+  products is not needed.
+- InnerSource usually deploys to a single instance. Then, the
+  needed scope for deployment training might be narrowed down.
+- Motivation: InnerSource is less exciting than open source, and the
+  pool of possible contributors is smaller. Motivation is more
+  challenging and needs more effort and skill.
+
+InnerSource-specific topics will need attention:
+- Transfer pricing doesn't apply to open source but tends to be a
+  barrier for corporations operating in different fiscal
+  jurisdictions.
+- Heavily outsourced development scenarios might require focusing
+  competency growth on product owners, if there are scant
+  developers inhouse.
+
+InnerSource supervision ought to focus on different questions:
+
+* Purpose: controlling the redundant developments and ensuring
+   InnerSource software is proactively managed.
+* Push/promote integrating InnerSource components and
+   contributing upstream.
+* Identify where InnerSource are de-facto or critical solutions
+   and assess suitability (avoid InnerSource monoliths).
+
+There's usually no need to curate enterprise-grade InnerSource
+software or to manage open source software development skills and
+resources (GGI activities 1.4 and 1.5).
+
+In exchange, software and documentation findability are
+challenging because public search engines usually don't work
+inside corporate borders.
+
+## Build trust around InnerSource
+
+Building trust means dealing with fears, uncertainty, and doubts
+(FUD). The usual sources for FUD are:
+
+  - **Legal context**, because most participants usually are
+    unaware of the relevant details.
+  - **Security**, mainly due to the not-invented here (MIH)
+    syndrome. In fact having the source code available ought to
+    help building trust by enabling auditability, and shifting
+    some testing from back-box to white-box.
+  - **Internal context**, specifically in organizations where the
+    InnerSource initiative sponsorship is limited.
+
+### Activities
+
+InnerSource has its specific legal challenges:
+ - InnerSource license. This might change in the future, but for
+   the moment, in contrast to open source, there are no public
+   InnerSource licenses for reference.
+ - Transfer pricing, for corporations operating in several fiscal
+   jurisdictions.
+ - Export control. The same is needed for any kind of software
+   licensing schema. But InnerSource has better options for control
+   than open source since it's privative software and all of its
+   development happens in-house.
+
+Managing vulnerabilities is slightly better under InnerSource, and
+way worse than in open source: There are dependencies on privative
+software and services (usually avoided in open source but quite usual
+in InnerSource).
+
+Dependency management needs to take care of both:
+ - Internal InnerSource dependencies
+ - Dependencies on privative software and services (usually
```

Comment:
What do you mean by `privative`?  I don't understand (even after looking up the word).

Reply:
Oops. This is a false friend: In spanish, _privativo_ is a stronger variant of _private_, a way to refer to [propietary software](https://www.gnu.org/proprietary/proprietary.en.html) stressing the fact that it deprives its users of certain rights.

Words like _propietary_ and _property_ stress ownership (control by the owner) while _private_ stresses deprivation (for others).

Would _private software_ work in english?

Reply:
Ah, no problem!  We’ll figure out the right wording!   What would be the difference “proprietary software” and “internal InnerSource software”?

Reply:
> stressing the fact that it deprives its users of certain rights.

Restricted?